tests/scripts/build_dgl.sh

```diff
@@ -10,8 +10,10 @@ fi
 if [[ $1 != "cpu" ]]; then
     CMAKE_VARS="$CMAKE_VARS -DUSE_CUDA=ON"
     if [[ $1 == "gpu" ]]; then
+        # CI is now running on g4dn instance.
         # Avoid error: Unknown CUDA Architecture Name 9.0a in CUDA_SELECT_NVCC_ARCH_FLAGS
         export TORCH_CUDA_ARCH_LIST=7.5
+        CMAKE_VARS="$CMAKE_VARS -DCUDA_ARCH_NAME=Turing"
```

Comment:
Including this line inside the if will make cugraph compile DGL for all architectures.

Reply:
I think the `-DCUDA_ARCH_NAME=Turing` addition can be made when we are not on the CPU so that the CI runtime for cugraph test is fast too.